src/Components/pages/Rocket.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets, reserveRocket } from '../../Redux/Rockets/Rockets';
+import RocketItem from '../RocketsComponents/RocketItem';
+import '../../App.css';
+
+function Rocket() {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const handleReserveRocket = (id) => {
+    dispatch(reserveRocket(id));
+  };
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+
```

Comment:
- [ ] [Optional] it would be nice to still see the rockets I reserved after I navigate out of rockets page and back, the `fetcRockets` dispatch could be called only on initial render

Reply:
Thanks for the notice @VICTOR-CHUKWU, I will work on that. 😄